yare-model-converters/src/main/resources/com/sabre/oss/yare/common/converter/aliases/typeAliases.properties

```diff
@@ -50,3 +50,4 @@ BigDecimal=java.math.BigDecimal
 List=java.util.List
 Map=java.util.Map
 Set=java.util.Set
+Undefined=com.sabre.oss.yare.core.model.Expression$Undefined
```

Comment:
I am wondering if something like `Expression.Undefined` would not be better here. `Undefined` is an inner class and personally I would find it easier to read with outer class name included. What do you think?

Reply:
@adamszadkowski good point. I will commit a change.